base_maintenance/views/maintenance_team_views.xml

```diff
@@ -19,15 +19,14 @@
         </field>
     </record>
 
-    <!-- There is no search view to inherit, so creating it-->
     <record id="maintenance_team_view_search" model="ir.ui.view">
```

Comment:
On 13.0 and 14.0 this view is no longer necessary :smile:

Reply:
For v14 it is removed in the migration, but for 13.0 we may have to leave it like this. Others might be inheriting from it and we can cause them some errors.